Assets/Scripts/UI/CatFactManager.cs

```diff
@@ -5,6 +5,7 @@
 public class CatFactManager : MonoBehaviour
 {
     [SerializeField] private GameObject _catFactPrefab;
+    public CatfactSeperation CatfactSeperation;
 
```

Comment:
you don't want to have the class name and the object itself have the exact same syntax. consider changing the object name to _catfactSeperation